srcDoc/command_line_parameters.rst

```diff
@@ -19,14 +19,14 @@ application, or both:
 
 As you will see in the tables, the ``-pub`` parameter specifies a
 publishing application and the ``-sub`` specifies a subscribing
-application. If you do not specify See ``-pub`` then ``-sub`` is
+application. If you do not specify See ``-pub``, then ``-sub`` is
```

Comment:
If you don't specify the ``-pub``option then ...

Reply:
Shall I make this change, @javmorcas, or do you want to?

Reply:
Since it is your branch, to avoid issues, could you make the changes? (If it is ok).